scala_proto/private/scala_proto_aspect.bzl

```diff
@@ -111,6 +121,7 @@ def _compile_sources(ctx, toolchain, proto, src_jars, deps):
         scalac = toolchain.scalac,
         dependency_info = legacy_unclear_dependency_info_for_protobuf_scrooge(ctx),
         unused_dependency_checker_ignored_targets = [],
+        stamp_target_label = stamp_label,
```

Comment:
Why not to pass `stamp_label` instead of `ctx.label`? I think in this context `ctx.label` is effectively unused because dependency analyser is disabled.

Reply:
`ctx.label` is unused, but I think `stamp_label` and `ctx.label` have different semantics. I can't think of very strong case against having just one, but let's say we want to log actual target while compiling scala. Maybe parameter `target_label` should be removed if it is not used and can be taken from ctx.